pkgs/development/compilers/hip/default.nix

```diff
@@ -160,10 +160,26 @@ in stdenv.mkDerivation (finalAttrs: {
     wrapProgram $out/bin/hipconfig.pl ${lib.concatStringsSep " " wrapperArgs}
   '';
 
-  passthru.updateScript = rocmUpdateScript {
-    name = finalAttrs.pname;
-    owner = finalAttrs.src.owner;
-    repo = finalAttrs.src.repo;
+  passthru = {
+    # All known and valid general GPU targets
+    # We cannot use this for each ROCm library, as each defines their own supported targets
+    # See: https://github.com/RadeonOpenCompute/ROCm/blob/77cbac4abab13046ee93d8b5bf410684caf91145/README.md#library-target-matrix
+    gpuTargets = lib.forEach [
+      "803"
+      "900"
+      "906"
+      "908"
+      "90a"
+      "1010"
+      "1012"
+      "1030"
```

Comment:
There is gfx11 as well now (RX 7xxx series). I’m not sure if ROCm supports it already though.
Does this list need to be hardcoded in nix?

Reply:
It's not removed from the repo, it's not in it yet.
That matrix is part of a PR in the works.

Reply:
Also ``hip.gpuTargets`` is really mainly for stuff like ``torch`` and ``tensorflow``, any of the libraries specify their own compatible devices and we build them all by default in nixpkgs. (Part of why some of these outputs are so... large.)